scripts/py/vcache_utils.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
```

Comment:
If you ever want it, I have the CudaStatTag class popped out in another branch

Reply:
good to know. let this be the start of a stats scripting directory in replicant? honestly, I just use It to reformat the csv into a table I can more easily manage in excel.

Reply:
Any scripts that are generally useful for manycore should go in manycore repository, but it seems like you're still iterating on this so I'm fine with this for now.

Reply:
well you know what they say about rules of thumb... some are made to be broken.

Reply:
you could make the argument that this more generally useful for cuda than for manycore.

Reply:
Wait, thumbs are meant to be broken?

You heard the discussion in meeting this morning. Replicant is a public repo but manycore is the focus repository. If it's generally useful, and it's finalized it should go in manycore.

Reply:
Yeah I heard that... but I think when it comes to data post-processing from an experiment I think most researchers are particular enough that they'll have their own flow anyway. That's why it's great that are stats infrastructure within the simulation just dumps out a CSV.

Reply:
SGTM. Then let's move the scripts directory inside bfs or gups since it's your flow at the moment

Reply:
No. This is shared code between the two benchmarks, I'll likely reuse it for more, and I think it's OK for replicant to have a common scripts directory.

Reply:
But if I can always make another submodule.

Reply:
well instead I just moved it into the one that already exists.